dotcom-rendering/src/components/TextBlockComponent.stories.tsx

```diff
@@ -104,7 +104,6 @@ export const QuotedDropCap = () => {
 		<div css={containerStyles}>
 			<TextBlockComponent
 				html={quotedHtml}
-				forceDropCap={false}
```

Comment:
As mentioned in the PR description, `forceDropCap: false` was previously equivalent to `forceDropCap: undefined`, and resulted in the normal rules being used. With this change, the way to get the normal rules to apply is to set `forceDropCap: undefined`, which in this case means dropping the prop.